### ISSUE TITLE
Question about the reported test accuracy

### ISSUE THREAD

**Issue by polaris-973** [MMorafah/PACFL#3]
Dear MMorafah:

Hello! Thanks for your scientific research achievement and open source efforts, I would like to ask whether the test accuracy reported in the paper is the initial accuracy obtained by directly testing cluster models in local data sets or the final accuracy after a local step training?

**Comment by MMorafah** [comment]
Hello,

Thanks for your interest in our work. It is the final accuracy after the local step training at the end of federation.

Regards,
Mahdi

**Comment by polaris-973** [comment]
Thanks for your timely reply!

**Comment by polaris-973** [comment]
Dear MMorafah:

Sorry to interrupt again, but I would like to confirm whether the accuracy of the FedAVG algorithm reported in the paper is the initial test accuracy(without local training step) or the final test accuracy? Thanks!

**Comment by MMorafah** [comment]
Hello,

It is final test accuracy of the global model.

Regards,
Mahdi
